### PR TITLE
community/zstd: upgrade to 1.3.8

### PR DESCRIPTION
**Pull request by OTLabs**

### CODE REVIEW

**Comment by OTLabs** [comment]
I assume that tests failed due to lack of space left on the drive:
```
**** flush write error test **** 
echo -e foo |  ../programs/zstd > /dev/full
zstd: error 25 : Write error : No space left on device (cannot write compressed block) 
echo -e foo |  ../programs/zstd |  ../programs/zstd -d > /dev/full
zstd: error 70 : Write error : No space left on device (cannot write decoded block) 
```

**Comment by andypost** [comment]
It looks makes sense to try cmake build https://github.com/facebook/zstd/wiki#makefile in follow-up

File: community/zstd/APKBUILD [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Contributor: stef <l0ls0fo2i@ctrlc.hu>
 # Maintainer: André Klitzing <aklitzing@gmail.com>
 pkgname=zstd
-pkgver=1.3.7
+pkgver=1.3.8
 pkgrel=1
```
Comment (andypost) [inline community/zstd/APKBUILD]: it should be started with 0

**Comment by OTLabs** [comment]
The binaries are created just fine, the problem is a huge test suite when huge input data files generated. If I disable check() the package builds without any problem.

**Comment by andypost** [comment]
Tested locally, it using more then 4GB while running tests
```
4,2G	community/zstd/
```
And data
```
*** zstd command line interface 64-bits v1.3.8, by Yann Collet ***
2240 fuzzer tests completed   med :3490 MB - Compressed : 327 MB => 9.40% 
./invalidDictionaries
./decodecorpus -t -T30
seed: 20143
298           :   2 MB - Consumed :4761 MB - Compressed : 447 MB => 9.40% 
/*stdin*\            :  9.40%   (5000000000 => 469887399 bytes, /*stdout*\)    
```

**Comment by andypost** [comment]
restart of CI helped

**Pull request closed without merging**